graph/telemetry/istio/appender/appender.go

```diff
@@ -186,6 +190,21 @@ func ParseAppenders(o graph.TelemetryOptions) (appenders []graph.Appender, final
 		}
 		appenders = append(appenders, a)
 	}
+	if _, ok := requestedAppenders[AmbientAppenderName]; ok || o.Appenders.All {
+		waypoints := defaultWaypoints
+		waypointsString := o.Params.Get("waypoints")
+		if waypointsString != "" {
+			var waypointsErr error
+			waypoints, waypointsErr = strconv.ParseBool(waypointsString)
+			if waypointsErr != nil {
+				graph.BadRequest(fmt.Sprintf("Invalid waypoints param [%s]", waypointsString))
+			}
+		}
+		a := AmbientAppender{
+			Waypoints: waypoints,
+		}
+		appenders = append(appenders, a)
+	}
```

Comment:
This also needs to move to the Finalizer section.  This should go, I think, after OutsiderAppender, and before the Health appender.  I'm not exactly sure, if these waypoint nodes are not getting the correct health then perhaps it should be run as the last finalizer.  But to start let's put it before health.

Reply:
Moved after the Outsider Appender.